lessons/module-2/github-collaboration.md

```diff
@@ -112,30 +112,17 @@ Be aware, this is just an example of how you may see issues organized in the wil
 
 Here are the ones that we want you to focus on when working on your project:
 
-##### Priority
-These labels are good for filtering issues - showing what makes sense for a person to tackle.
+##### Music
+https://www.youtube.com/watch?v=cYM3QfpoHAw
 
-Often you will also see priority levels listed on these labels with `P1` for priority 1, `P2` for priority 2, etc.
+##### More Music
+https://www.youtube.com/watch?v=ANzgMYgoJsA
```

Comment:
Since we have a music header, consider grouping the links together in a bullet list.